### PR TITLE
Test consistency

### PR DESCRIPTION
**Pull request by DaddyWesker**
Ok, so I've added test consistency. Currently it is failing on the last line of the test_consistency method since `getNearEntities` returns Oak Log instead of oak_log (which is looks like this everywhere else). So, it needs some changes i guess.  I'll think about it.

Also there are some changes to visualization - now it works in the main thread. 

This PR won't works only with my [Vereya PR ](https://github.com/trueagi-io/Vereya/pull/45)

### CODE REVIEW

**Review by CICS-Oleg — COMMENTED** [review]
Why do you think it's unnecessary?

File: mcdemoaux/vision/vis.py
Comment (CICS-Oleg) [inline mcdemoaux/vision/vis.py:26]: Why do you think it's unnecessary?

Comment (CICS-Oleg) [reply mcdemoaux/vision/vis.py:26]: Or do we have now new format and new conversion with new image transfer protocol?

Comment (DaddyWesker) [reply mcdemoaux/vision/vis.py:26]: @CICS-Oleg  well, I'm not sure actually. Does we need that if-statement? Currently bgr2rgb done in timestamped_video_frame in my PR https://github.com/trueagi-io/minecraft-demo/pull/136/files which still waiting for your approval =)

**Comment by DaddyWesker** [comment]
So, I've deleted here all my changes related to visualizer. So now this PR is only about test_consistency.

File: tagilmo/utils/vereya_wrapper.py
Comment (DaddyWesker) [reply tagilmo/utils/vereya_wrapper.py]: @Necr0x0Der  I've changed that one. Though it now done through for loop and I'm not sure it is better solution than three replace for one string.

File: tagilmo/utils/vereya_wrapper.py
Comment (noskill) [inline tagilmo/utils/vereya_wrapper.py:245]:
```suggestion
```

File: tests/vereya/test_consistency.py
Comment (noskill) [inline tests/vereya/test_consistency.py:11]: if this function has nothing specific you can replace it with init_mission from common.py

Comment (DaddyWesker) [reply tests/vereya/test_consistency.py:11]: @noskill  it does differ

File: tagilmo/utils/vereya_wrapper.py
Comment (Necr0x0Der) [inline tagilmo/utils/vereya_wrapper.py:451]: Why only one alternative of ingredients is considered?
Also, I'm not sure if it is really better to explicitly take the structure of receipts into account, or to use a generic function like this:
```
    def remove_mcprefix_rec(data):
        if isinstance(data, str):
            return data.split('.')[-1] if 'minecraft' in data else data
        if isinstance(data, dict):
            r = {}
            for k in data.keys():
                r[remove_mcprefix_rec(k)] = remove_mcprefix_rec(data[k])
            return r
        if isinstance(data, list):
            return remove_mcprefix_rec(l) for l in data]
        return data
```

Comment (DaddyWesker) [reply tagilmo/utils/vereya_wrapper.py]: @Necr0x0Der works like a charm. Thanks. I've added to PR

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**
